src/main/resources/resources/logisim/strings/gui/gui_de.properties

```diff
@@ -27,64 +27,64 @@ showStateDialogTitle = Zustand für %s
+# ==> stepCoarse = Coarse-grained summary
+# ==> stepCoarseDescription = Data is captured only when all signals have stabilized, ignoring any transient signal fluctuations that may happen during propagation. Each captured value is recorded as if it were held stable for %s.
+# ==> fineDetail = Capture fine-grained details
+# ==> stepFineDescription = Data is captured whenever any signal value changes, including any transient signal fluctuations that may happen during propagation. Transient fluctuations are recorded as if the gate delay were %s, and stable signals are recorded as if they were held stable for %s.
+realTime = Durchgehender Echtzeit-Modus
+# ==> realCoarse = Coarse-grained summary
+# ==> realCoarseDescription = Daten werden kontinuirlich gespeichert, wenn der Simulator eingeschaltet ist, but transient signal fluctuations are ignored. Jede Sekunde wird als %s der Simulierten Schaltungszeit gespeichert.
+# ==> realFine = Fine-grained details
+# ==> realFineDescription = Data is captured continuously, whenever the simulator is enabled, includin transient signal fluctuations. Each second of real time is recorded as %s of simulated circuit time.
+clockTime = Takt-Modus
+# ==> clockCoarse = Coarse-grained summary
+# ==> clockCoarseDescriptionDual = Daten werden bei jeder Takt Flanke gespeichert, ignoring any transient fluctuations that may happen between transitions or during propagation. Each clock cycle is recorded as if it took %s per cycle (%s per tick).
+# ==> clockCoarseDescriptionRising = Daten werden bei jeder steigenden Takt Flanke gespeichert, ignoring any transient fluctuations that may happen between transitions or during propagation. Each clock cycle is recorded as if it took %s per cycle (%s per tick).
+# ==> clockCoarseDescriptionFalling = Daten werden bei jeder fallenden Takt Flanke gespeichert, ignoring any transient fluctuations that may happen between transitions or during propagation. Each clock cycle is recorded as if it took %s per cycle (%s per tick).
+# ==> clockCoarseDescriptionHigh = Daten werden kontinuirlich gespeichert, wenn der Takt Pegel hoch ist. Änderungen werend der Pegel niedrig ist werden ignoriert. Changes when the clock is high are recorded as if the gate delay were %s, and each clock cycle is recorded as if it took %s per cycle (%s per tick).
+# ==> clockCoarseDescriptionLow = Daten werden kontinuirlich gespeichert, wenn der Takt Pegel niedrig ist. Änderungen werend der Pegel hoch ist werden ignoriert. Changes when the clock is low are recorded as if the gate delay were %s, and each clock cycle is recorded as if it took %s per cycle (%s per tick).
+# ==> clockFine = Fine-grained details
+# ==> clockFineDescription = Daten werden gespeichert, wenn sich ein Signal ändert. , including any transient signal fluctuations that may happen during propagation. Transient fluctuations are recorded as if the gate delay were %s, and each clock cycle is recorded as if it took %s per cycle (%s per tick).
```

Comment:
Incomplete translation

Reply:
I started working on these, but sadly don't have the time to finish them at the moment.
I don't want to delay this PR any further.

Reply:
I agree, localisation can be further improved with later PRs.